### PR TITLE
Issue fixed #6445

### LINKED ISSUE #6445: There should not be a warning message if the user is trying to remove the item from the cart
**Bagisto Version:** Master

**Steps To reproduce**--

**1.** **Log** in as a customer or **visit** the shop.
**2.** Click on any product.
**3.** Add the same product into the **cart**.
**4.** Click on the **mini** cart.
**5.** Click on the cross icon **multiple** times for **removing** the product from the **cart**.

**Expected result:** There should not be a **warning** message if the user is trying to **remove** the item from the **cart** 

**Actual result:** There is a **warning** message if the user is clicking on the **cross** icon **multiple** times for **removing** the item from the **cat**. Please find the screenshot [Here](https://prnt.sc/wNmqNb7rbMIK)

### PR DESCRIPTION
**Pull request by akathuria289**
## Issue Reference
#6445

### CODE REVIEW

**Comment by Sanjaybhattwebkul** [comment]
@akathuria289 
Please resolved the **conflicts**

**Comment by akathuria289** [comment]
@Sanjaybhattwebkul  Resolved

File: packages/Webkul/Velocity/src/Http/Controllers/Shop/CartController.php [hunk 1]
```diff
@@ -95,12 +95,15 @@ public function addProductToCart()
...
+
             session()->flash('warning', __($exception->getMessage()));
 
             $product = $this->productRepository->find($id);
 
-            Log::error('Velocity CartController: ' . $exception->getMessage(),
-                ['product_id' => $id, 'cart_id' => cart()->getCart() ?? 0]);
+            Log::error(
+                'Velocity CartController: ' . $exception->getMessage(),
+                ['product_id' => $id, 'cart_id' => cart()->getCart() ?? 0]
```
Comment (jitendra-webkul) [inline packages/Webkul/Velocity/src/Http/Controllers/Shop/CartController.php:105]: Break array into multiple lines

**Comment by akathuria289** [comment]
Done

**Comment by devansh-webkul** [comment]
I am seeing old commits in this PR.

**Pull request closed without merging**